index.html

```diff
@@ -248,6 +248,11 @@ <h2><a href="https://www.pixelmator.com/pro/">Pixelmator</a></h2>
             <em>by <a href="https://www.pixelmator.com/blog/">Pixelmator Team</a></em>
             <p>An image editor for Mac and iOS designed to make the most powerful professional image editing tools accessible to everyone</p>
           </li>
+          <li>
+            <h2><a href="https://www.maildesigner365.com/">Mail Designer 365</a></h2>
+            <em>by <a href="https://www.pixelmator.com/blog/">TOWER ONE GmbH</a></em>
```

Comment:
This URL was copied and pasted from the previous entry. It should be:

`<em>by <a href="https://tower-one.net/">TOWER ONE GmbH</a></em>`

Reply:
Doh, thanks I've fixed it in https://github.com/good/openweb/pull/42/commits/1b44ffa7b01c1bc03f3cadeee420fbc22d370ef1